tests/integration/components/paper-item-test.js

```diff
@@ -0,0 +1,182 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+import Ember from 'ember';
+import wait from 'ember-test-helpers/wait';
+
+const { set, get } = Ember;
+
+moduleForComponent('paper-item', 'Integration | Component | paper item', {
+  integration: true
+});
+
+test('single action checkboxes should react to checkbox clicks', function(assert) {
+  assert.expect(1);
+  set(this, 'checkboxEnabled', false);
+  this.render(hbs`
+    {{#paper-list}}
+      {{#paper-item}}
+        <p>Checkbox 1</p>
+        {{paper-checkbox class="md-secondary" value=checkboxEnabled onChange=(action (mut checkboxEnabled))}}
+      {{/paper-item}}
+    {{/paper-list}}
+  `);
+  return wait().then(() => {
```

Comment:
@miguelcobain When paper-item initializes it runs the proxiable-mixin `init` hook. it had the `run.scheduleOnce('afterRender')` call to `registerProxy` method which would take care of proxies when they are already in DOM. the problem was that after the properties have setup in this run loop we would modify them again and the deprecation would trigger. And it also left some place for ambiguity as `didInsertElement` was running only once and did the DOM manipulation code for each proxy even though they might be not in the DOM just yet. So we had to provide the initial context for paper-item in a template, otherwise the buttons would not initialized properly ever as `didInsertElement` would run just once.